### PR TITLE
rework on plugin

### PR DESCRIPTION
**Pull request by TesDevelopment**

### CODE REVIEW

**Review by pozm — CHANGES_REQUESTED** [review]
actually fix it,m it deosnt work properly

**Comment by TesDevelopment** [comment]
> actually fix it,m it deosnt work properly

que?

File: src/lua/admin.plugin.lua [hunk 1]
```diff
@@ -1,33 +1,91 @@
...
+                    Clone.Visible = true
+                    Clone.Parent = Commands.Frame.List
+                end
+            elseif debug then
+                Utils.Notify(LocalPlayer,"No commands in plugin",string.format("Plugin %s has no commands. Skipping.",plugin.Name),5)
+            end
+        end)
+        if not ss then
+            if debug then
+                math.randomseed(os.time())
```
Comment (stangithuboffical) [inline src/lua/admin.plugin.lua:66]: MUTIPLY THE TIME!!!!

File: src/lua/admin.plugin.lua [hunk 2]
```diff
@@ -1,33 +1,91 @@
...
+            end
+        end
+    elseif debug then
+        Utils.Notify(LocalPlayer,"Plugin not loaded.",string.format("Plugin %s was not loaded as it is on the disabled list.",plugin.Name),5)
+    end
+end
+
+if config.pluginsenabled then
+    for i, file in pairs(listfiles("fates_plugins")) do
+        LoadPlugin(loadstring(readfile(file))())
```
Comment (stangithuboffical) [inline src/lua/admin.plugin.lua:79]: USE LOAD!!!!!!!

File: src/lua/admin.plugin.lua [hunk 3]
```diff
@@ -1,33 +1,91 @@
...
+    end
+elseif debug then
+    Utils.Notify(LocalPlayer,"Plugins disabled.","You have disabled all plugins.",5)
+end
+
+AddCommand("refreshplugins",{"rfp","refresh","reload"},"Loads all new plugins.",{}, function(caller)
+    for i, file in pairs(listfiles("fates_plugins")) do
```
Comment (stangithuboffical) [inline src/lua/admin.plugin.lua:86]: LISTFILES ISNT DEFINED!!!!

**Review by stangithuboffical — CHANGES_REQUESTED** [review]

**Comment by ViniDalvino** [comment]
LOL

File: src/lua/admin.plugin.lua (hunk 1 above)
Comment (ViniDalvino) [reply src/lua/admin.plugin.lua:66]: COCK AND BALL

File: src/lua/admin.plugin.lua (hunk 3 above)
Comment (ViniDalvino) [reply src/lua/admin.plugin.lua:86]: PENIS

**Comment by ViniDalvino** [comment]
pee come from the ball

File: src/lua/admin.plugin.lua (hunk 3 above)
Comment (stangithuboffical) [reply src/lua/admin.plugin.lua:86]: OK I JUST ADDED LISTFILES TO SYNAPSE

**Comment by pozm** [comment]
bad code

**Pull request closed without merging**